plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/DependencyResolver.java

```diff
@@ -57,7 +60,8 @@
       String groupId, String artifactId, String version, IProgressMonitor monitor)
           throws CoreException {
 
-    final Artifact artifact = new DefaultArtifact(groupId + ":" + artifactId + ":" + version);
+    final Artifact artifact =
```

Comment:
Seems like `final` is unnecessary starting with Java 8.

Reply:
done